python/py-pastel/Portfile

```
@@ -5,13 +5,16 @@ PortGroup           python 1.0
 
 name                py-pastel
 version             0.2.1
-revision            0
+revision            1
```

```suggestion
revision            0
```

Thanks @dgilman for the PR. This shouldn't need a revision bump, since anyone who has successfully installed the port don't need to rebuild it.

The buildbots will attempt to rebuild the port without the revbump (such as here https://github.com/macports/macports-ports/commit/651df5ed3939844da92a23633772e5d2422982d5)